pkg/batchutils/batchutils.go

```diff
@@ -0,0 +1,28 @@
+package batchutils
+
+import (
+	"github.com/Masterminds/squirrel"
+	"github.com/loopfz/gadgeto/zesty"
+
+	"github.com/ovh/utask/db/sqlgenerator"
+	"github.com/ovh/utask/models/task"
+)
+
+// States in which the task won't ever be run again
+var FinalStates = []string{task.StateDone, task.StateCancelled, task.StateWontfix}
+
+// RunningTasks returns the amount of running tasks sharing the same given batchId.
+func RunningTasks(dbp zesty.DBProvider, batchId int64) (int64, error) {
+	query, params, err := sqlgenerator.PGsql.
+		Select("count (*)").
+		From("task t").
+		Join("batch b on b.id = t.id_batch").
```

Comment:
`id_batch` should be `batch_id` to be coherent with the rest of the code, and it reads better.

Reply:
I couldn't agree more, but `id_batch` is the column's name in the database, I have no power over it sadly. :smiling_face_with_tear:
https://github.com/ovh/utask/blob/master/sql/schema.sql#L44